src/Symfony/Component/Form/Form.php

```diff
@@ -11,12 +11,12 @@
 
 namespace Symfony\Component\Form;
 
-use Symfony\Component\Form\Exception\RuntimeException;
```

Comment:
Ah, this was done automatically by PhpStorm. Will revert this.

I think we should always keep all use statements ordered in all branches. This makes merging much easier in general. StyleCI, for example, does that automatically.